src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,21 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int weight;
+
     public Bulldozer() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    public int getWeight() {
+        return weight;
+    }
+
+    public void setWeight(int weight) {
```

Comment:
Are you using this setter somewhere?

Reply:
Common mistakes to this task says: Don’t forget about access modifiers for your custom fields in Excavator, Truck and Bulldozer classes.
You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters).

Reply:
Ok, I agree with you. My bad